### ISSUE TITLE
Review the alt text for Donate page images and ensure that they adhere to WCAG

### ISSUE THREAD

**Issue by IsaacDesigns** [hackforla/website#2568]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the Donate page.

#### Details
Developers identified that some alt texts in the Donate page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
UI/UX Design
- [x] Make a copy of the tab Template [Page Name]- [Status] 
- [x] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [x] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [x] For each image on the Donate page identify if the alt text needs to be changed, or not, using the recourses below, and fill out the google sheet.
#### Summary
If images provide more meaning = needs alt text
If images are decorative = null alt text (alt="")
- [x] When completed change your google sheet tabs status to Audit Complete
- [x] Review with Design Lead or another designer/researcher
- [x] Mark issue as done

Development
- [ ]  Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix

### Resources/Instructions
[Donate page](https://www.hackforla.org/donate/)
[Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[Decorative Images and alt text](https://www.w3.org/WAI/tutorials/images/decorative/)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
[Google sheet Tab for your page](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit?pli=1#gid=14599403)

**Comment by iilashi** [comment]
Progress: Waiting to review with team
Blockers: None
Availability:
ETA: Complete

**Comment by IsaacDesigns** [comment]
Hi @ilashi great job auditing the Donate page!

**Comment by JessicaLucindaCheng** [comment]
Dev leads need to write another issue to update donate page images' alt text since QA reviewed it. So, double check the requested changes in the spreadsheet match what was implemented in #2886. If it isn't, write an issue for devs to fix it.

**Comment by JessicaLucindaCheng** [comment]
@JessicaLucindaCheng is writing the dev issues for this.

**Comment by JessicaLucindaCheng** [comment]
@JessicaLucindaCheng is finished writing the dev issues for this. So, removing the `ready for dev lead` label and moving this to the Done column.

To address the following:
> Dev leads need to write another issue to update donate page images' alt text since QA reviewed it. So, double check the requested changes in the spreadsheet match what was implemented in #2886. If it isn't, write an issue for devs to fix it.

Issue #3238 was written to fix an alt text that still need to be fixed. The rest of the alt text were implemented correctly in issue #2886

**Comment by blulady** [comment]
> @JessicaLucindaCheng is finished writing the dev issues for this. So, removing the `ready for dev lead` label and moving this to the Done column.
> 
> To address the following:
> 
> > Dev leads need to write another issue to update donate page images' alt text since QA reviewed it. So, double check the requested changes in the spreadsheet match what was implemented in #2886. If it isn't, write an issue for devs to fix it.
> 
> Issue #3238 was written to fix an alt text that still need to be fixed. The rest of the alt text were implemented correctly in issue #2886

Replaced both of these issues with #3630
